src/TodoGroup/TodoGroup.js

```diff
@@ -1,20 +1,27 @@
 import React from "react";
 import PropTypes from "prop-types";
+import { connect } from "react-redux";
 import TodoItem from "../TodoItem/TodoItem";
-import AddTodo from "../AddTodo/AddTodo";
+
 import "./styles.css";
+import AddTodo from "../AddTodo/AddTodo";
+
+function TodoGroup(props) {
+   const { color, created, title, id, todoItems, nextItemId, todoCheckBox } = props;
 
-function TodoGroup({ todoList, color, created, title, id, addTodo, handleCheckbox }) {
-   const todoItems = todoList.map((item, index) => {
-      if (item["filtered"] && item.filtered !== false) {
+   const todos = todoItems.map((item) => {
+      if (
+         (item.sortByDone === true || item.sortByDone === null) &&
+         (item.sortByTitle === null || item.sortByTitle === true)
+      ) {
          return (
             <TodoItem
-               key={index}
+               key={item.itemId}
```

Comment:
Use id prop. from todoList item. Not 'itemId'

Reply:
There is no 'itemId' on todoGroup object. Please check 'warning' or 'error' messages before commit or making pull request.